homeassistant/components/camera/opencv.py

```diff
@@ -0,0 +1,100 @@
+"""
+Component that performs OpenCV processes on images.
```

Comment:
1. I was planning on building out a stand alone camera (not really sure benefit of it now that I think about it)
2. Yup copy paste - will update